static/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx

```diff
@@ -273,24 +275,43 @@ class GroupEventDetails extends Component<Props, State> {
                   mostRecentActivity as GroupActivityReprocess
                 )}
                 {this.renderContent(eventWithMeta)}
-              </div>
-              <div className="secondary">
+              </StyledLayoutMain>
+              <StyledLayoutSide>
                 <GroupSidebar
                   organization={organization}
                   project={project}
                   group={group}
                   event={eventWithMeta}
                   environments={environments}
                 />
-              </div>
+              </StyledLayoutSide>
             </Fragment>
           )}
-        </div>
+        </StyledLayoutBody>
       </div>
     );
   }
 }
 
+const StyledLayoutBody = styled(Layout.Body)`
+  /* Makes the borders align correctly */
+  padding-top: 0 !important;
+  padding-bottom: 0 !important;
+  /* Layout Footer alignment fix */
+  margin-bottom: -20px;
+`;
+
+const StyledLayoutMain = styled(Layout.Main)`
+  @media (min-width: ${p => p.theme.breakpoints[2]}) {
+    border-right: 1px solid ${p => p.theme.border};
+    padding-right: ${space(4)};
+  }
+`;
+
+const StyledLayoutSide = styled(Layout.Side)`
+  padding-top: ${space(3)};
+`;
+
```

Comment:
Again, I hate all of this. But for now it’s necessary because this page has a bunch of custom borders that divvy up each section. I will delete them.